Reason: avoid unnecessary generation of documentation if not wanted
Source: Daniel Mierswa <impulze@impulze.org>
Upstream: no

diff -Naur libmp3splt-0.7.orig/configure.ac libmp3splt-0.7/configure.ac
--- libmp3splt-0.7.orig/configure.ac	2011-07-14 12:45:21.000000000 +0200
+++ libmp3splt-0.7/configure.ac	2011-08-27 23:33:06.000000000 +0200
@@ -269,14 +269,28 @@
 
 # Check for doxygen, so we can decide if we add it to the "doc" 
 # make target.
+AC_ARG_ENABLE(doxygen, [AC_HELP_STRING([--disable-doxygen],[ Disable documentation generation. ]) ],
+    [enable_doxygen=$enableval],[enable_doxygen="yes"])
+
+if test "x$enable_doxygen" = xyes;then
+
 AC_PATH_PROG([DOXYGEN],[doxygen])
+
+fi
+
 AC_ARG_VAR(DOXYGEN,
            [Location of the 'doxygen' program; if found we can generate documentation for the C code])
 AM_CONDITIONAL(HAVE_DOXYGEN, [ test "x$DOXYGEN" != "x" ])
 
 # Check for dot, the tool that makes the nice little caller graphs for the
 # doxygen documentation.
+
+if test "x$enable_doxygen" = xyes;then
+
 AC_PATH_PROG([GRAPHVIZ],[dot])
+
+fi
+
 AC_ARG_VAR(GRAPHVIZ,
            [Location of the 'dot' program from the graphviz suite; if found we can generate caller graphs for the documentation of the C code])
 AM_CONDITIONAL(HAVE_GRAPHVIZ, [ test "x$GRAPHVIZ" != "x" ])
